Cargo.toml

```
@@ -65,6 +66,7 @@ packed_simd = { version = "0.3.4", package = "packed_simd_2", features = ["into_
 [features]
 default = ["alloc"]
 alloc = ["zeroize/alloc"]
+group = ["group_crate", "rand_core"]
```

With #485 landed this can now be:

```suggestion
group = ["dep:group", "rand_core"]
```